tests/kafkatest/services/kafka/kafka.py

```diff
@@ -277,6 +278,9 @@ def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAI
         self.isolated_controller_quorum = None # will define below if necessary
         self.configured_for_zk_migration = False
 
+        if use_new_coordinator is None :
+            self.use_new_coordinator = self.context.globals.get["use_new_coordinator", False]
```

Comment:
get is a method, use ()
```suggestion
            self.use_new_coordinator = self.context.globals.get("use_new_coordinator", False)
```